addons/website/models/website_controller_page.py

```diff
@@ -45,9 +52,33 @@ def _check_user_has_model_access(self):
     def _compute_name_slugified(self):
         for rec in self:
             if not rec.model_id or not rec.page_type:
+                rec.name_slugified = False
                 continue
             rec.name_slugified = slugify(rec.page_name or '')
 
+    def _inverse_name_slugified(self):
+        for rec in self:
+            rec.name_slugified = slugify(rec.name_slugified)
```

Comment:
Not sure I get it... both compute and inverse do assign the same field ?

Reply:
yes. in the spec, the field has to be writable. But since it is a string that should be url-compatible, the server needs to clean it, hence this inverse.
@ryv-odoo confirms that, as weird as it is, it is a feature and won't trigger infinite loops